.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -123,6 +123,10 @@ stages:
           inputs:
             artifactName: 'vhd-release-notes'
             targetPath: 'release-notes.txt'
+        - task: PublishPipelineArtifact@0
+          inputs:
+            artifactName: 'vhd-image-bom'
+            targetPath: 'image-bom.json'
```

Comment:
The build logs show that `image-bom.json` has been published and uploaded successfully. Is there way to view that file separately, for this build, at this stage?

Reply:
you can download it from the pipeline artifacts from PR builds :)